content/de/cloud_native_security.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cloud Native Security
+status: Feedback appreciated
+category: Konzept
+---
+
+## Was es ist
+
+Cloud Native Security beschreibt den Ansatz, Konzepte aus dem Bereich Cyber Security in [Cloud Native Anwendungen](/cloud_native_apps/) zu integrieren. 
```

Comment:
This change will lead to https://glossary.cncf.io/cloud-native-apps/ (an English term). 😊

```suggestion
Cloud Native Security beschreibt den Ansatz, Konzepte aus dem Bereich Cyber Security in [Cloud Native Anwendungen](/cloud-native-apps/) zu integrieren. 
```